docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,192 @@
+|`updated` |Float |The last time the timeline was updated, using a
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the timeline.
+|`version` |String |The timeline's version.
+|==============================================
+
+[[col-obj]]
+[discrete]
+==== columns object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`aggregatable` |Boolean |Indicates whether the field can be aggregated on all
+indices.
+|`category` |String |The ECS field set to which the field belongs.
+|`description` |String |Column field description.
+|`example` |String |Column field example.
+|`indexes` |String |The list of indices where this field has the same type, or
+null if all indices have the same type for the field.
+|`id` |String |ECS field alias or user defined field name.
+|`name` |String |Reserved for future use, typically `null`.
+|`searchable` |Boolean |Indicates whether this field is indexed for search on
+all indices.
+|`type` |String |???
+|==============================================
+
+[[dataProvider-obj]]
+[discrete]
+==== dataProviders object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`id` |String |???
+|`name` |String |???
+|`enabled` |Boolean |???
+|`excluded` |Boolean |???
+|`kqlQuery` |String |???
+|`queryMatch` |QueryMatchInput |???
+|`and` |dataProviders |???
+|==============================================
+
+[[eventNotes-obj]]
+[discrete]
+==== eventNotes object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`created` |Float |The time the note was created, using a 13-digit Epoch
+timestamp.
+|`createdBy` |String |The user who added the note.
+|`eventId` |String |The ID of the event to which the note was added.
+|`note` |String |The note's text.
+|`noteId` |String |The note's ID
+|`timelineId` |String |The ID of the timeline to which the note was added.
+|`updated` |Float |The last time the note was updated, using a
+13-digit Epoch timestamp.
+|`updatedBy` |String |The user who last updated the note.
+|`version` |String |The note's version.
+|==============================================
+
+[[favorite-obj]]
+[discrete]
+==== favorite object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`favoriteDate` |Float |The time the timeline was marked as a favorite.
+|`fullName` |String |The full name of the user who marked the timeline as
+a favorite.
+|`keySearch` |String |???
+|`userName` |String |The {kib} username of the user who marked the
+timeline as a favorite.
+|==============================================
+
+[[filters-obj]]
+[discrete]
+==== filters object
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`exists` |String |???
+|`meta` |FilterMetaTimelineInput |???
```

Comment:
<img width="677" alt="Screenshot 2020-06-26 at 10 35 58" src="https://user-images.githubusercontent.com/6295984/85843271-deda9380-b798-11ea-8268-fd1eef419db9.png">

alias:
disabled: whether the filter is temporarily disabled
field:
key:
negate:
params: if it is a phrase type of query, we keep the query value here
<img width="849" alt="Screenshot 2020-06-26 at 11 28 50" src="https://user-images.githubusercontent.com/6295984/85848025-5b24a500-b7a0-11ea-82ad-52175c9a9183.png">

type: phrase, range, custom
value: appears if it is a custom type of query.

Reply:
@angorayc - are all these filter types relevant for timeline?

Reply:
@angorayc - just for my general knowledge, why is the first example a `custom` query type and not `range`?